cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -33,6 +33,10 @@ data:
   # Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.
   # CSI_LOG_LEVEL: "0"
 
+  # By default OMAP generator sidecar is deployed with CSI provisioner pod, to disable
+  # it set it to false.
+  # CSI_ENABLE_OMAP_GENERATOR: "false"
```

Comment:
Enabling a new sidecar in a rook patch release by default is a bit higher risk. Should we consider not enabling by default for 1.5.x, and then enable by default in 1.6? Or is it really preferred to enable by default and it's worth the risk?

Reply:
It should not affect any functionality as we have required RBAC already present in the rook 1.5. I have disabled it by default for now.